backend/src/govsg/services/govsg.service.ts

```diff
@@ -212,12 +228,22 @@ export async function processSingleRecipientCampaign(
       where: { campaignId },
       transaction,
     })
-    await GovsgMessage.create({
-      campaignId,
-      recipient: data.recipient,
-      languageCode,
-      params: data,
-    } as GovsgMessage)
+    const govsgMessage = await GovsgMessage.create(
+      {
+        campaignId,
+        recipient: data.recipient,
+        languageCode,
+        params: data,
+      } as GovsgMessage,
+      { transaction, returning: true }
+    )
+    await GovsgVerification.create(
```

Comment:
nit: It looks to me like both the `create` calls are related to each other. Do you think it would be beneficial to make use of a [nested create](https://sequelize.org/docs/v6/advanced-association-concepts/creating-with-associations/)?

(Sorry if I didnt pick this up the previous time I reviewed this PR.)

Reply:
I can try this out in the next PR if that's ok.